OMCompiler/Compiler/NBackEnd/NBVariable.mo

```diff
@@ -563,6 +583,13 @@ public
         fail();
       end try;
     end getVarSafe;
+
+    function size
+      input VariablePointers variables;
+      output Integer i;
+    algorithm
+      i := ExpandableArray.getNumberOfElements(variables.varArr);
```

Comment:
Do we want to count array-variables as one variable or according to their dimensions?

Reply:
In the use case where i implemented it i wanted it that way (for correctly sizing hash tables) but we probably need size with considering array sizes too